### PR TITLE
test: Introduce elm-test package

### PR DESCRIPTION
**Pull request by mxinden**
Adding two basic (fuzz-) tests for Utils.Filter.parseMatcher. Including
tests in Travis runs. Adding make target "test" and "dev-server".

### CODE REVIEW

File: ui/app/tests/elm-package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "version": "1.0.0",
+    "summary": "Sample Elm Test",
+    "repository": "https://github.com/user/project.git",
+    "license": "BSD-3-Clause",
```
Comment (stuartnelson3) [inline ui/app/tests/elm-package.json]: We should update these fields. License is Apache-2.0.

**Comment by mxinden** [comment]
@stuartnelson3 Updated PR.

**Comment by mxinden** [comment]
@stuartnelson3 I can't find out what the issue is with *fossa.io*. To login I have to grant *fossa* full read and write access to all my *GitHub* repositories.

@fabxc Can you help us here? Can you see what the license complain is all about?

**Comment by stuartnelson3** [comment]
Seems there are some conflicts now, too

**Pull request merged**